lib/widgets/screens/explore/explore.dart

```diff
@@ -62,35 +115,76 @@ class _ExploreCardScrollState extends State<ExploreCardScroll> {
     final ThemeData theme = Theme.of(context);
     final AppLocalizations l10n = AppLocalizations.of(context)!;
 
+    return FutureBuilder(
+      future: _recommendedUsers,
+      builder: (context, snapshot) => AppUtility.widgetForAsyncSnapshot(
+        snapshot: snapshot,
+        onReady: () => Column(
+          children: [
+            ..._createCards(snapshot.data?.response ?? []),
+            TextButton(
+                onPressed: () {
+                  setState(_loadMoreRecommendations);
```

Comment:
setState fails to execute here because it cannot contain an async function. _loadMoreRecommendations() gets executed, but the widget does not rebuild so you cannot see the new results. You do not need to make changes here if you address my other comment.